src/async/stompest/async/client.py

```diff
@@ -135,14 +135,16 @@ def session(self):
     # STOMP commands
     #
     @defer.inlineCallbacks
-    def connect(self, headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None):
-        """connect(headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None)
+    def connect(self, headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None, certKey=None, privateKey=None):
+        """connect(headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None, certKey=None, privateKey=None)
```

Comment:
Ok: if `context.load_cert_chain()` has been called with a certfile and keyfile, then these will be attributes on the context itself.  The context is accessible via the `Stomp.config.sslContext` hierarchy.  You can see this around line `stompest/async/client.py:62`.  So what is needed is a way for the `clientFromString()` call inside the `endpointFactory()` function to be able to access the required cert and key parameters on the config.  Currently the config seems to not be accessible inside `endpointFactory()`.  Is that correct @nikipore?

I would much prefer accessing the config object than passing arguments through multiple calls to get them where they need to go.